docs/hoon/twig/wut-test/bar-or.md

```diff
@@ -10,13 +10,25 @@ sort: 9
 ## Expands to
 
 ```
+:loop
 :ifno  p  
```

Comment:
At the very least, it's adjacent to the expansion: https://github.com/urbit/arvo/blob/master/arvo/hoon.hoon#L7319.

That raises the overall question: what purpose should these the expansions in the docs serve? I think, to the degree it's possible, they should be valid twigs with the actual behavior of the twig in question.